pkg/apis/core/v1/types.go

```diff
@@ -863,3 +863,17 @@ type (
 func (a Archive) IsEmpty() bool {
 	return len(a.Literal) == 0 && len(a.URL) == 0
 }
+
+func (fn Function) GetConcurrency() int {
+	if fn.Spec.Concurrency == 0 {
+		return 500
```

Comment:
Please define constants above and use them here.

DefaultConcurrency and DefaultRequestPerPod